.github/workflows/dusk_ci.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: nightly
+          toolchain: nightly-2020-10-25
```

Comment:
For `fmt` is fine, but for the testing or any other purpose toolchains I think we should use lates nightly.

Reply:
I'm not really sure why `fmt` is being installed in the test case. Since it's @ZER0 's code, I think he should elaborate.